src/pages/UploadPage.tsx

```diff
@@ -111,24 +126,21 @@ export default function UploadPage() {
         <DropZone setFile={setFile} />
       </Box>
       <Box sx={{ display: 'flex', flexDirection: 'row', mb: 30 }}>
-        <Button
+        <LoadingButton
           size="medium"
           color="secondary"
           variant="contained"
           component="label"
           sx={{ width: '100px' }}
           onClick={sendFiles}
           disabled={isSendButtonDisabled} // Disable the button conditionally
+          loading={isLoading}
           disableElevation
         >
           Send
-        </Button>
+        </LoadingButton>
       </Box>
-      {alert ? (
-        <Alert severity="error">Oops, something went wrong... Try again later.</Alert>
-      ) : (
-        <></>
-      )}
+      {alert ? <Alert severity="error">{errorMessage}</Alert> : <></>}
       {success ? <Alert severity="success">🎉 Files sent successfully!</Alert> : <></>}
```

Comment:
Not your fault, but could you do me a favour and change this to singular, since we are only sending one file? Thanks 😁